source/getting-started.html.md

```diff
@@ -0,0 +1,238 @@
+  ...
+}
+```
+
+The `document_type` and `schema_name` fields are used to define the format of
+the data represented in a [`ContentItem`][content-item]. The main fields that
+are affected by this are [`details`](#details) and [`links`](#links).
+
+The `schema_name` refers to a file referenced in
+[GOV.UK Content Schemas][content-schemas-repo] which utilises
+[JSON Schema][] to describe the rules of the data. The `document_type` defines
+the name of the format on GOV.UK. Often these fields share the same value as
+there is often only one schema for a particular `document_type`.
+
+This is the case with [Vat Rates][vat-rates-api] where the content returned
+utilises the [`answer` schema][answer-schema].
+
+### Details
+
+```
+{
+  ...
+  "details": {
+    "body": "\n<div class=\"highlight-answer\">\n<p>The standard <abbr title=\"Value Added Tax\">VAT</abbr> rate is <em>20%</em></p>\n</div>\n\n<h2 id=\"vat-rates-for-goods-and-services\">\n<abbr title=\"Value Added Tax\">VAT</abbr> rates for goods and services</h2>\n\n<table>\n  <thead>\n    <tr>\n      <th>Rate</th>\n      <th>% of <abbr title=\"Value Added Tax\">VAT</abbr>\n</th>\n      <th>What the rate applies to</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <td>Standard</td>\n      <td>20%</td>\n      <td>Most goods and services</td>\n    </tr>\n    <tr>\n      <td>Reduced rate</td>\n      <td>5%</td>\n      <td>Some goods and services, eg children’s car seats and home energy</td>\n    </tr>\n    <tr>\n      <td>Zero rate</td>\n      <td>0%</td>\n      <td>Zero-rated goods and services, eg most food and children’s clothes</td>\n    </tr>\n  </tbody>\n</table>\n\n<p>The standard rate of <abbr title=\"Value Added Tax\">VAT</abbr> increased to 20% on 4 January 2011 (from 17.5%).</p>\n\n<p>Some things are exempt from <abbr title=\"Value Added Tax\">VAT</abbr>, eg postage stamps, financial and property transactions.</p>\n\n<p>The <a href=\"/vat-businesses/vat-rates\"><abbr title=\"Value Added Tax\">VAT</abbr> rate businesses charge</a> depends on their goods and services.</p>\n\n<p>Check the <a href=\"https://www.gov.uk/rates-of-vat-on-different-goods-and-services\">rates of <abbr title=\"Value Added Tax\">VAT</abbr></a> on different goods and services.</p>\n\n",
+    "external_related_links": []
+  },
+  ...
+}
+```
+
+The `details` field of a [`ContentItem`][content-item] is an object that
+contains data in a particular structure relevant for the type of content. This
+structure is described by the [schema](#document-type-and-schema-name), which
+itself is defined by the `schema_name` field.
+
+In the case of [Vat Rates][vat-rates-api] an [`answer` schema][answer-schema]
+is used. The `answer` schema  allows a `body` value, which contains the HTML
+for the page and a `external_related_links` value, which is an array that can
+be used to provide links to associated pages hosted off GOV.UK.
+
+### Links
+
+The `links` field is used to describe relationships between items of content.
```

Comment:
I think for consistency we should call it "content items" instead of "items of content".

Reply:
Good point. I wanted to try avoid "content items" as it collides with "ContentItem" but it's probably not good to use "items of content" I think just switching this to: "The `links` field is used to describe relationships between content" works though?